coe/cloud-adoption.html

```diff
@@ -1,75 +1,38 @@
 ---
-layout: default
```

Comment:
Rather than needing to copy and paste HTML for future centers, this sort of Jekyll markdown layout means you can just copy the structure below and insert new content into it.